dbt_project.yml

```diff
@@ -279,9 +279,36 @@ models:
     addresses_events:
       +schema: addresses_events
       +materialized: table
+      arbitrum:
+        +schema: addresses_events_arbitrum
+        +materialized: table
+      avalanche_c:
+        +schema: addresses_events_avalanche_c
+        +materialized: table
+      bnb:
+        +schema: addresses_events_bnb
+        +materialized: table
       ethereum:
         +schema: addresses_events_ethereum
         +materialized: table
+      fantom:
+        +schema: addresses_events_fantom
+        +materialized: table
+      gnosis:
+        +schema: addresses_events_gnosis
+        +materialized: table
+      optimism:
+        +schema: addresses_events_optimism
+        +materialized: table
+      polygon:
+        +schema: addresses_events_polygon
+        +materialized: table
+    addresses_events_testnets:
+      +schema: addresses_events_testnets
+      +materialized: table
+      goerli:
+        +schema: addresses_events_testnets_goerli
+        +materialized: table
```

Comment:
We don't really want to specify `materialized: table` here as it's overwritten in the config blocks. 
I believe with this current setup the top level `addresses_events.first_funded_by` is actually materialized as a table, while it should just be a view of all the sub models.